README.md

```diff
@@ -92,48 +90,38 @@ After you're finished please remove all the comments and instructions!
     <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
   </ul>
 </details>
-
+ -->
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+
+- **[Displays the header section of a mobile view]**
+- 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
-## 🚀 Live Demo <a name="live-demo"></a>
+<!-- ## 🚀 Live Demo <a name="live-demo"></a>
 
 > Add a link to your deployed project.
 
 - [Live Demo Link](https://yourdeployedapplicationlink.com)
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
 
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
```

Comment:
- Hey, let's improve this section by adding some descriptive instructions on how the users can have your project run locally on their machine to make the readme more professional.
- Please, kindly remove the test section as you won't be adding any test on this project. see the image below to update them:

![Screenshot from 2023-02-21 13-05-11](https://user-images.githubusercontent.com/59397159/220340777-d004c42b-51ee-4c4a-8ac4-b57b64bd1b81.png)

Reply:
@zieeco I removed the run test part of the Getting started